io.github.Fontforge.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.Fontforge
+branch: stable
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+command: fontforge
+
+finish-args:
+  - --socket=x11
```

Comment:
socket=x11 is not enough to use the app on x11. See the docs.

Reply:
hm. which documentation are you referring to? The only wiki page containing "x11" I could find was this: https://github.com/flathub/flathub/wiki/App-Requirements and it doesn't mention anything like that flag not being enough.

Reply:
It needs IPC as well.

Reply:
k. added.